include/core/network.h

```diff
@@ -11,18 +11,21 @@
 #include <stdint.h>
 #include <contiki-net.h>
 
-/** Halts the process until UDP data has been sent */
+/** Halts the process until UDP/TCP data has been sent */
```

Comment:
Can you split this into two separate comments for both macros? This is for Doxygen to properly annotate the macros, otherwise it'll have this comment only for the first one.